lib/linguist/languages.yml

```diff
@@ -3738,12 +3738,12 @@ Ruby:
   - Deliverfile
   - Fastfile
   - Gemfile
-  - Gemfile.lock
```

Comment:
Why? It's not a ruby file? It's obviously gonna be part of a ruby project, but the file itself is not a ruby file (I used this list for rubocop).

I guess it's good for statistics about the repo but not for actual ruby files detection?

(See `Gemfile.lock` file search: https://github.com/search?utf8=%E2%9C%93&q=Gemfile.lock&type=Commits&ref=searchresults)